packages/stream-tests/src/__tests__/invalid_tip.test.ts

```diff
@@ -62,7 +62,10 @@ describe('Test loading a stream when pubsub replies with an invalid tip', () =>
       if (message.typ == MsgType.QUERY && message.stream.equals(streamID)) {
         const tipMap = new Map().set(streamID.toString(), nonExistentTip)
         const response = { typ: MsgType.RESPONSE, id: message.id, tips: tipMap }
-        ipfs2.pubsub.publish(ceramic.pubsubTopic, serialize(response))
+        ipfs2.pubsub.publish(ceramic.pubsubTopic, serialize(response)).catch((error) => {
```

Comment:
For a test, seems like this might be better with `await expect(...).resolves.not.toThrow()`

Reply:
agreed, but the problem is that the callback passed to `pubsub.subscribe` is expected to be a serial, non-async function, so we can't use `await` here.

Reply:
I don't love it either, wish there was a better way